staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go

```diff
@@ -310,3 +216,21 @@ func printSingleResourceUsage(out io.Writer, resourceType v1.ResourceName, quant
 		fmt.Fprintf(out, "%v", quantity.Value())
 	}
 }
+
+func printPodResourcesSum(out io.Writer, total v1.ResourceList, withNamespace, printContainers bool, columnWidth int) {
```

Comment:
`withNamespace` and `printContainers` are unused.

Reply:
Fixed